src/Forms/AWRequiredFields.php

```diff
@@ -11,12 +12,21 @@
  *
  * @author Russell Michell russell@silverstripe.com
  * @package advancedworkflow
+ *
+ * @deprecated 5.4.0 Will be renamed to Symbiote\AdvancedWorkflow\Forms\AWRequiredFieldsValidator
  */
 class AWRequiredFields extends RequiredFields
 {
     protected $data = array();
     protected static $caller;
 
+    public function __construct()
+    {
+        $message = 'Will be renamed to Symbiote\\AdvancedWorkflow\\Forms\\AWRequiredFieldsValidator';
```

Comment:
May need to update depending on https://github.com/silverstripe/silverstripe-userforms/pull/1354#discussion_r1870393853 regarding namespace vs just class name

Reply:
Will leave as is